### ISSUE TITLE
Fix gradient hexadecimal color in chat. [1.20.4 or lower]

### ISSUE THREAD

**Issue by jagermeizter** [Brikster/Chatty#280]
For some reason the hexadecimal gradient colors are not working in the chat. With the Luckperms plugin I have tried to set this prefix "**{#FFFFFF:#9EBA2E &lUSER}**" and it doesn't work. I have also tried to use the "**%luckperms_prefix**" variant and it does not work in the plugin configuration (format: ' '). The prefix color is white and does not change to another color. In 1.19 it worked perfectly. Please, if you could update your plugin by solving this error or perhaps others, you would solve the problem for many. I appreciate the time to read this, I send you my regards!

PS: _An apology for my English generated by google translate._

**Comment by c1oudychan** [comment]
You can use Chatty v3 + MiniMessage for it

**Comment by jagermeizter** [comment]
> You can use Chatty v3 + MiniMessage for it

How do I make that work?
Is it a plugin that I should download and add or how do I do it?
I'm sorry for my ignorance.

**Comment by c1oudychan** [comment]
[Chatty v3 (click for download)](https://github.com/Brikster/Chatty/actions/runs/7783247524/artifacts/1219976791) (you can download it yourself from [Actions tab](https://github.com/Brikster/Chatty/actions)). It uses [MiniMessage](https://docs.advntr.dev/minimessage/format.html) format for messages. 
old config (v2) will be moved to another folder bc v3 uses different config.

**Comment by jagermeizter** [comment]
> [Chatty v3 (click for download)](https://github.com/Brikster/Chatty/actions/runs/7783247524/artifacts/1219976791) (you can download it yourself from [Actions tab](https://github.com/Brikster/Chatty/actions)). It uses [MiniMessage](https://docs.advntr.dev/minimessage/format.html) format for messages. old config (v2) will be moved to another folder bc v3 uses different config.

Bro, is it normal for the plugin to weigh 25 MB? The last one published on spigot only weighs 500 kb.

**Comment by c1oudychan** [comment]
Yes, because all libraries in jar & this is different version (you can see description on [page](https://github.com/Brikster/Chatty/tree/v3))

**Comment by jagermeizter** [comment]
> Sí, porque todas las bibliotecas están en jar y esta es una versión diferente (puedes ver la descripción en [la página](https://github.com/Brikster/Chatty/tree/v3) )

I understand and I thank you. I removed all the settings from v2 and am now using v3. The plugin works correctly and takes the color codes perfectly.

Thanks for helping, greetings!
